src/cljx/racehub/util/time.cljx

```diff
@@ -154,14 +166,24 @@
    (time/interval (time/epoch)
                   (timestamp-to-datetime ts))))
 
+(s/defn unix-time :- UnixTime
```

Comment:
These two are different- one returns the current unix time (no conversion), the other one is a conversion. So no round-tripping here. Just wrappers over to-long and from-long, from the cljs-time.coerce ns